src/Microsoft.IdentityModel.Protocols/Microsoft.IdentityModel.Protocols.csproj

```diff
@@ -25,7 +25,7 @@
   <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.4'">
     <PackageReference Include="System.Collections.Specialized" Version="4.3.0-*" />
     <PackageReference Include="System.Diagnostics.Contracts" Version="4.3.0-*" />
```

Comment:
should these also use updated latest version?

Reply:
These are not affected by the vulnerability and are already at the latest version (4.3.0).